src/commands/schema/pull.ts

```diff
@@ -13,7 +13,13 @@ export default class PullSchemaCommand extends SchemaCommand {
       default: false,
     }),
     staged: Flags.boolean({
-      description: "Pulls staged schema instead of the active schema",
+      description:
+        "Pulls the staged schema. Not applicable when there is no staged schema.",
```

Comment:
This description is misleading IMO. I think the intuition we want developers to have is that there is effectively always an active and staged schema pointer, but they can point to the same thing.

This flag should be the default.